content/en/guides/models/track/runs/filter-runs.md

```diff
@@ -11,26 +11,30 @@ Use your project page to gain insights from runs logged to W&B. You can filter a
 
 ## Filter runs
 
-Filter runs based on their status, tags, or other properties with the filter button.
+Filter runs based on their status, [tags]({{< relref "#filter-runs-with-tags" >}}), [regular expressions (RegEx)]({{< relref "#filter-runs-with-regular-expressions-regex"  >}}) or other properties with the filter button.
 
+See Customize run colors for more information on how to [edit, randomize, and reset run colors]({{< relref "guides/models/track/runs/run-colors" >}}).
 
 ### Filter runs with tags
 
-Filter runs based on their tags with the filter button.
+Filter runs based on their tags with the filter button. 
 
-{{< img src="/images/app_ui/filter_runs.gif" alt="" >}}
-
-
-### Filter runs with regex
+1. Click on the **Runs** tab from the project sidebar.
+2. Select the **Filter** button (looks like a funnel) at the top of the runs table.
+3. From left to right, select `"Tags"` from the dropdown menu, a logic operator, and a filter search value.
 
-If regex doesn't provide you the desired results, you can make use of [tags]({{< relref "tags.md" >}}) to filter out the runs in Runs Table. Tags can be added either on run creation or after they're finished. Once the tags are added to a run, you can add a tag filter as shown in the gif below.
+{{< img src="/images/app_ui/filter_runs.gif" alt="" >}}
 
-{{< img src="/images/app_ui/tags.gif" alt="If regex doesn't provide you the desired results, you can make use of tags to filter out the runs in Runs Table" >}}
+### Filter runs with regular expressions (RegEx)
 
+1. Click on the **Runs** tab from the project sidebar.
+2. Click on the search box at the top of the runs table.
+3. Ensure that the **RegEx** toggle (.*) is enabled (the toggle should be blue).
+4. Enter your regular expression in the search box.
 
 ## Search runs
 
-Use [regex](https://dev.mysql.com/doc/refman/8.0/en/regexp.html) to find runs with the regex you specify. When you type a query in the search box, that will filter down the visible runs in the graphs on the workspace as well as filtering the rows of the table.
+Use regular expressions ([RegEx](https://dev.mysql.com/doc/refman/8.0/en/regexp.html)) to find runs with the regular expression you specify. When you type a query in the search box, that will filter down the visible runs in the graphs on the workspace as well as filtering the rows of the table.
```

Comment:
This looks a little funny with "regular expressions" not being part of the link.